_commands/tools/scp.md

```diff
@@ -0,0 +1,33 @@
+---
+---
+
+scp
+-------
+
+`scp` is used to make a _secure copy_ of files or directories to and from a remote host. Similar to "cp" except either the source or destination should be a remote host. You'll want to run this command from your local machine.
+
+~~~ bash
+$ scp source user@host:destination
```

Comment:
Another cool thing here is that this uses your SSH config. E.g. on my machine I frequently do

    scp caen:~/Desktop/file.c ~/Desktop/myproj